pysb/tests/test_sensitivity_analysis.py

```diff
@@ -121,21 +125,91 @@ def test_create_png(self):
                                            'test3_P_H_P_prime.png'))
 
     def test_unique_simulations_only(self):
+        vals = [.8, .9, 1.1, 1.2, 1.3]
+        sens = PairwiseSensitivity(
+            values_to_sample=vals,
+            objective_function=self.obj_func_cell_cycle,
+            observable=self.observable,
+            solver=self.solver
+        )
+        sens.run()
+        self.sens.create_plot_p_h_pprime(save_name='test4',
+                                         out_dir=self.output_dir)
+        assert os.path.exists(os.path.join(self.output_dir,
+                                           'test4_P_H_P_prime.png'))
+
+    def test_param_pair(self):
+        vals = [.9, 1.0, 1.1]
+        sens = PairwiseSensitivity(
+            values_to_sample=vals,
+            objective_function=self.obj_func_cell_cycle,
+            observable=self.observable,
+            solver=self.solver,
+            sample_list=['k1', 'cdc0']
+        )
+        sens.run()
+        self.sens.create_plot_p_h_pprime(save_name='test4',
+                                         out_dir=self.output_dir)
+        assert os.path.exists(os.path.join(self.output_dir,
+                                           'test4_P_H_P_prime.png'))
+
+    def test_all_params(self):
+        vals = [.9, 1.1]
+
+        sens = PairwiseSensitivity(
+            values_to_sample=vals,
+            objective_function=self.obj_func_cell_cycle,
+            observable=self.observable,
+            solver=self.solver,
+            sens_type='all'
+        )
+        sens.run()
+        self.sens.create_plot_p_h_pprime(save_name='test4',
+                                         out_dir=None)
+        assert os.path.exists('test4_P_H_P_prime.png')
+
+    @raises(Exception)
```

Comment:
Use a more specific exception class. Otherwise, unrelated errors can end up getting missed. Likewise for the other tests in this file.